ninja-core/src/main/java/ninja/WrappedContext.java

```diff
@@ -44,7 +44,7 @@ public WrappedContext(Context wrapped) {
 
     @Override
     public String getRequestUri() {
-        return wrapped.getRequestUri();
```

Comment:
please leave it that way - it's intentionally referencing the deprecated version.